File: PerfTester-Xcode/PerfTester-Xcode/Models/Helpers/Dog.swift
//
//  Dog.swift
//  PerfTester-Xcode
//
//  Created by Patryk Średziński on 24.06.2018.
//  Copyright © 2018 private.perftesterxcode. All rights reserved.
//

import UIKit
import RealmSwift

class Dog: Object {

    @objc dynamic var id: Int = 0
    @objc dynamic var name: String = ""
    @objc dynamic var age: Int = 0
    @objc dynamic var owner: Person?
    
}

File: PerfTester-Xamarin/PerfTester-Xamarin/Models/Helpers/Dog.cs

using System;
using Realms;

namespace PerfTesterXamarin.Models.Helpers
{
    public class Dog : RealmObject
    {
        public Int32 Id { get; set; }
        public string Name { get; set; }
        public int Age { get; set; }
        public Person Owner { get; set; }

    }

}
